setup-dynamodb-local.mjs

```diff
@@ -0,0 +1,29 @@
+import { mkdir } from 'fs/promises'
+import { existsSync } from 'fs'
+import { execSync } from 'child_process'
+
+const dynamodbDir = 'dynamodb_local_latest'
+const dynamodbArchiveURL =
+  'https://s3.us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest.tar.gz'
+
+// Check if the dynamodbDir directory exists
+async function runSetup() {
+  if (!existsSync(dynamodbDir)) {
+    // If it doesn't exist, create it
+    await mkdir(dynamodbDir, { recursive: true })
+
+    // Download and extract DynamoDB Local
+    try {
+      console.log('Downloading DynamoDB Local...')
+      execSync(`curl -L ${dynamodbArchiveURL} | tar -xz -C ${dynamodbDir}`, {
+        stdio: 'inherit',
+      })
+      console.log('DynamoDB Local setup completed successfully.')
+    } catch (error) {
+      console.error('Error setting up DynamoDB Local:', error.message)
+      process.exit(1)
```

Comment:
I don't know that you need to catch and log the exception --- just let it throw.